sample/dynamicfeature/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
I assume this is added by mistake?

Reply:
> I assume this is added by mistake?

Not mistake. Same with sample/lib/.gitignore.
It's created by AndroidStudio.